Documentation/running-docker-images.md

```diff
@@ -6,12 +6,12 @@ rkt features native support for fetching and running Docker container images.
 
 To reference a Docker image, use the `docker://` prefix when fetching or running images.
 
-Note that Docker images do not support signature verification, and hence it's necessary to use the `--insecure-skip-verify` flag.
+Note that Docker images do not support signature verification, and hence it's necessary to use the `--insecure-options=all` flag.
```

Comment:
Yeah this is going to be the last step, checking each use and reference to usage and make sure it's accurate. Wanted to make sure the general approach is ok before going through with that.